ui/src/app/views/workspace-share/component.ts

```diff
@@ -175,9 +175,11 @@ export class WorkspaceShareComponent implements OnInit {
       this.autocompleteNoResults = true;
       return;
     }
-    this.userService.user(this.searchTerm).subscribe((response) => {
-      this.autocompleteLoading = false;
+    const searchTerm = this.searchTerm;
+    this.userService.user(searchTerm).subscribe((response) => {
+      if (this.searchTerm !== searchTerm) { return; }
```

Comment:
This code gets executed later when the API response comes back. At this point, `searchTerm` is unchanged as it refers to the value in the closure (and the same value we sent for the request), however `this.searchTerm` may be different if the user has input some different value.

Reply:
Thank you - this is helpful